src/openforms/emails/digest.py

```diff
@@ -179,3 +214,73 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_failed_registration_backends() -> list[FailedRegistrationBackend]:
+    registration_backends = FormRegistrationBackend.objects.all()
+    form_backend_mappings = [
+        {registration_backend.form.admin_name: registration_backend}
+        for registration_backend in registration_backends
+        if registration_backend.form.is_available
+    ]
+
+    failed_registration_backends = []
+    for mapping in form_backend_mappings:
+        for form_name, registration_backend in mapping.items():
+            form = registration_backend.form
+            backend_options = registration_backend.options
+            backend_type = registration_backend.backend
+            plugin = register[backend_type]
+
+            # errors in general configuration are more straightforward for the user,
+            # so we show the exact ones
+            if broken_general_config := check_general_config(plugin):
+                failed_registration_backends.append(
+                    FailedRegistrationBackend(
+                        config_name=plugin.verbose_name,
+                        exception_message=broken_general_config,
+                        form_name=form_name,
+                    )
+                )
+
+            # errors in the form level can be more detailed so here we want to show
+            # a more general error and the link to the broken form to investigate
+            if not is_form_config_valid(plugin, backend_options):
+                failed_registration_backends.append(
+                    FailedRegistrationBackend(
+                        config_name=plugin.verbose_name,
+                        exception_message=_(
+                            "Invalid registration backend configuration has been detected in the form level"
+                        ),
+                        form_name=form_name,
+                        form_id=str(form.id),
+                        form_level=True,
+                    )
+                )
+
+    return failed_registration_backends
+
+
+def collect_failed_logic_rules() -> list[FailedLogicRule]:
+    logic_for_active_forms = FormLogic.objects.filter(form__active=True)
```

Comment:
Same as above, I think the iterating logic can be improved and we need to take into account static/user defined variables. We can discuss this tomorrow